core/src/main/java/io/paradoxical/cassieq/admin/resources/api/v1/AccountResource.java

```diff
@@ -226,14 +206,12 @@ public Response addNewKey(
     @ApiResponses(value = { @ApiResponse(code = 204, message = "Ok"),
                             @ApiResponse(code = 500, message = "Server Error") })
     public Response deleteAccount(@StringTypeValid @PathParam("accountName") final AccountName accountName) {
+
+        lookupAccount(accountName);
```

Comment:
Not sure I like throwing web exceptions since they are runtime based.  Its not super clear that this is doing an account validation